thoth/storages/graph/dgraph.py

```diff
@@ -763,6 +763,22 @@ def retrieve_unsolved_python_packages(self, solver_name: str) -> dict:
 
         return self._postprocess_retrieve_packages(result)
 
+    def retrieve_not_analyzed_python_package_version_entities(self) -> List[dict]:
+        """Retrieve a list of package names, versions and index urls that were not analyzed yet by package-analyzer."""
+        query = """{
+            f(func: has(%s), first: 10) @filter(NOT has(%s)) {
```

Comment:
I'm returning only the first 10 results. Should we change it or make it dynamic?

Reply:
Yes, please see the linked example above.